### PR TITLE
[WNMGDS-1579] Add visual regression testing for child design systems

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

https://jira.cms.gov/browse/WNMGDS-1579

I spent too much time trying to make a command that would both build a DS statically and then take update snapshots with the static storybook. I still can't get webpack to have correct relative paths to fonts and images. Instead, you have to continue to run the storybook server separately and then run one of the new loki commands.

I have yet to add them to the CI tests

## How to test

Start up `yarn storybook`, `yarn storybook:healthcare`, or `yarn storybook:medicare` in one terminal window, and then run the corresponding new `yarn loki:update*` command in another. Images shouldn't change. You can also run `yarn loki test` if you add the right environment variable

If the GitHub diff page crashes because it's trying to load all the images, try this link with the images filtered out https://github.com/CMSgov/design-system/pull/1679/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&show-viewed-files=true

### CODE REVIEW

**Comment by pwolfert** [comment]
If the diff page crashes because it's trying to load all the images, try this link with the images filtered out https://github.com/CMSgov/design-system/pull/1679/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&show-viewed-files=true

**Comment by pwolfert** [comment]
I have a script update being tested over here https://ci.backends.cms.gov/wds/blue/organizations/jenkins/design-system%2Ftests/detail/pwolfert%2Fvrt-child-design-systems/10/pipeline/, but I'm still struggling to get the fonts to show up in the static version of the medicare storybook, so I get a bunch of errors when I run the loki tests for medicare

**Comment by pwolfert** [comment]
Okay, here it is working with a bandaid fix for now: https://ci.backends.cms.gov/wds/blue/organizations/jenkins/design-system%2Ftests/detail/pwolfert%2Fvrt-child-design-systems/11/pipeline

**Comment by pwolfert** [comment]
Built the storybook for this latest commit so I could use it for comparison: http://design-system-demo.s3-website-us-east-1.amazonaws.com/pwolfert/vrt-child-design-systems/mgov/storybook/

File: packages/ds-medicare-gov/src/components/Navbar/Navbar.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  title: 'Components/Navbar',
+  component: Navbar,
+  argTypes: {
+    className: {
+      control: { type: 'text' },
+      type: { name: 'string', required: false },
+    },
+  },
+  args: {
+    children: (
```
Comment (pwolfert) [inline packages/ds-medicare-gov/src/components/Navbar/Navbar.stories.jsx:17]: I took this code from one of the applications that uses it and modified it

File: packages/ds-medicare-gov/src/components/Navbar/Navbar.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      </>
+    ),
+    className: '',
+    inverse: true,
+  },
+};
+
+const Template = ({ data, ...args }) => <Navbar {...args} />;
+
+export const BasicNavbar = Template.bind({});
```
Comment (forrestbaer) [inline packages/ds-medicare-gov/src/components/Navbar/Navbar.stories.jsx:41]: Currently we use 'default' as the naming convention for this context, can change Basic to Default for these 3?

Everything else looks good!

Comment (pwolfert) [reply packages/ds-medicare-gov/src/components/Navbar/Navbar.stories.jsx]: @forrestbaer, I've updated the stories with https://github.com/CMSgov/design-system/pull/1679/commits/86603fe91da10cf7c29b556ff8f565ebc134369b

**Review by forrestbaer — APPROVED** [review]
👍🏻

**Pull request merged**